src/components/UserInfoCard.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <CardV2 class="flex flex-col items-center w-full gap-[1rem]">
+    <Identity
+      :class="{ 'cursor-pointer hover:scale-105 transition-transform': $listeners['click-avatar'] }"
+      :avatar-url="avatarSrc"
```

Comment:
should also set displayname here?

Reply:
You mean `alt`?

Reply:
Yes it is for `alt` but I think the prop is called `displayName`

Reply:
Updated